src/Common/Handlers/NewEnvironmentHandler.cs

```diff
@@ -132,7 +133,24 @@ bool terminateOtherVersionsRemoveVolumes
                         await _customerPortalClient.TerminateObjects<List<CustomerEnvironment>, CustomerEnvironment>(customerEnvironmentsToTerminate, terminateOperationAttibutes);
 
                         // wait until they're terminated
-                        await _environmentDeploymentHandler.WaitForEnvironmentsToBeTerminated(customerEnvironmentsToTerminate);
+                        List<long> ceTerminationFailedIds = await _environmentDeploymentHandler.WaitForEnvironmentsToBeTerminated(customerEnvironmentsToTerminate);
+
+                        if (ceTerminationFailedIds?.Count > 0)
+                        {
+                            string failedIdsString = string.Join(", ", ceTerminationFailedIds.Select(x => x.ToString()));
+                            string errorMessage = $"Stopping deploy process because termination of other environment versions failed. The environments with the Ids {failedIdsString} failed.";

```

Comment:
Adding list to the end will make logs more "similar" when scrolling
```suggestion
                            string errorMessage = $"Stopping deploy process because termination of other environment versions failed. Environment Ids of failed terminations: {failedIdsString}.";
```